app/build.gradle

```diff
@@ -1,5 +1,7 @@
 plugins {
```

Comment:
Why did you create directory "app"?

You don't need it. Everything from directory "app" should be in root folder